nestkernel/event.h

```diff
@@ -827,13 +843,20 @@ class GapJEvent : public SecondaryEvent
   static void
   set_syn_id( const synindex synid )
```

Comment:
Is this method still called from anywhere? I not, we should remove it. If yes, we should replace the call by a call to `add`syn_id()`.